samples/java/hibernate/src/main/java/com/google/cloud/postgres/CustomValueGenerator.java

```diff
@@ -0,0 +1,36 @@
+/* 
+ * Hibernate, Relational Persistence for Idiomatic Java
+ * 
+ * JBoss, Home of Professional Open Source
+ * Copyright 2014 Red Hat Inc. and/or its affiliates and other contributors
+ * as indicated by the @authors tag. All rights reserved.
+ * See the copyright.txt in the distribution for a
+ * full listing of individual contributors.
+ *
+ * This copyrighted material is made available to anyone wishing to use,
+ * modify, copy, or redistribute it subject to the terms and conditions
+ * of the GNU Lesser General Public License, v. 2.1.
+ * This program is distributed in the hope that it will be useful, but WITHOUT A
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ * PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public License,
+ * v.2.1 along with this distribution; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+ * MA  02110-1301, USA.
+ */
+package com.google.cloud.postgres;
+
+import java.time.LocalDateTime;
+import org.hibernate.Session;
+import org.hibernate.tuple.ValueGenerator;
+
+/**
+ * @author Brett Meyer
+ */
+public class CustomValueGenerator implements ValueGenerator<LocalDateTime> {
```

Comment:
nit: Could we rename this class to something like `CurrentLocalDateTimeGenerator`?

Reply:
Done